spot_driver/launch/spot_image_publishers.launch.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2023-2024 Boston Dynamics AI Institute LLC. All rights reserved.
+
+import os
+from enum import Enum
+from typing import List
+
+import launch
+import launch_ros
+from launch import LaunchContext, LaunchDescription
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.conditions import IfCondition
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+
+from spot_driver.launch.spot_launch_helpers import spot_has_arm
+
+
+class DepthRegisteredMode(Enum):
+    DISABLE = (0,)
+    FROM_SPOT = (1,)
+    FROM_NODELETS = (2,)
```

Comment:
These are likely supposed to be numbers not tuples. Also, can you add a `__repr__` function?